.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+	"presets": ["@wordpress/babel-preset-default"],
+	"plugins": [
```

Comment:
I figured out why we need it in the first place:

https://github.com/parcel-bundler/parcel/blob/060db2e2c56f08e223e9a9075035f0998e249763/packages/core/parcel-bundler/src/transforms/babel/jsx.js#L40-L49

Parcel tries to be smart and overrides our configuration.